app/views/layouts/_navigation.html.erb

```diff
@@ -0,0 +1,49 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light" id="mainNav">
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+    <a class="navbar-brand js-scroll-trigger" href="/" id="homepage">ObituaryHub</a>
+      <div class="collapse navbar-collapse" id="navbarResponsive">
+          <ul class="navbar-nav ml-auto">
+            <% if current_user %>
+              <li class="nav-item">
+                <button class="btn btn-outline-secondary mr-1"><a href="/obituaries/new">Add Obituary</a></button>
+              </li>
+              <li class="nav-item" id="logout">
+                <button class="btn btn-outline-secondary" type="button"><%= link_to "Logout", "/logout", method: :delete %></button>
+              </li>
+            <% else %>
+            <li class="nav-link">
+              <a class="nav-link" href="/users/new">Sign Up</a>
+            </li>
+            <li class="nav-link">
+              <a class="nav-link" href="/login">Login</a>
+            </li>
+            <% end %>
+          </ul>
+      </div>
+</nav>
+
+<nav class="navbar navbar-expand-lg navbar-light bg-light" id="secondaryNav">
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarText" aria-controls="navbarText" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="navbarText">
+    <ul class="navbar-nav mr-auto">
+      <li class="nav-item">
+        <a class="nav-link" href="/obituaries">All Obituaries</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="/obituaries/covid-19">COVID-19 Obituaries</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="/obituaries/recent">Recently Published</a>
+      </li>
+    </ul>
+    <form class="form-inline">
+      <a class="nav-link" href="/search/advanced">Advanced Search Options</a>
+      <input class="form-control mr-sm-2" type="search" placeholder="Search by Name" aria-label="Search">
+      <button class="btn btn-sm btn-outline-primary my-2 my-sm-0 text-dark" type="submit">Search</button>
+    </form>
+</div>
+</nav>
```

Comment:
Good idea moving this to a partial. It's crazy how much space the code for a navbar takes.

Reply:
Dont fully understand how to implement them yet, but partial seem like a real time saver.